Dockerfile-alpine

```diff
@@ -0,0 +1,60 @@
+# The MIT License
+#
+#  Copyright (c) 2015, CloudBees, Inc.
```

Comment:
Needs to be changed unless it is a full copy-paste

Reply:
Licence updated